docs/sources/writing-guide/contribute-documentation/index.md

```diff
@@ -54,21 +54,45 @@ Small changes might include:
 
 > **Note:** You can only edit a topic that is part of the latest release of documentation.
 
-## Before you begin
+### Before you begin
 
 - Create a [GitHub](https://www.github.com) account.
 
 **To edit a topic:**
 
 1. From a topic on the documentation website, click **Edit this page** (pencil icon).
-2. Enter your changes.
-3. Change the branch name, if required.
+1. Enter your changes.
+1. Change the branch name, if required.
    The branch name is auto-populated.
 
-4. Click **Propose changes**.
+1. Click **Propose changes**.
 
    A PR is created and then goes through the PR review and approval workflow.
 
+## Develop a new topic
+
+If you want to develop a new topic from scratch, you can create a documentation plan and collaborate with a member of the Docs team. According to the book [_Docs for Developers_](https://docsfordevelopers.com/), a documentation plan is a _flexible outline_ for anticipating where the writing process will lead you.
+
+Your documentation plan helps you to:
+
+- Identify existing information gaps, and explain how you will fill them.
+- Get feedback from users and stakeholders before the writing process begins.
+- Consider different approaches you might take, and decide on one of them.
+
+> **Note:** Only create new topics that are part of the upcoming release of documentation.
```

Comment:
This is a weird restriction to place on anyone writing new content.  Once the release is out, the feature can never be documented?   Again, I am worried that this will discourage documentation contributions. There are PLENTY of places where released features and code still needs to be documented and we can use all the help we can get.

Reply:
Right, it's not the best advice. I've deleted it in the most recent version of this PR.